internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -651,6 +672,33 @@ func setupVarRun(logger *log.Logger) error {
 	return nil
 }
 
+// SetupSystemDirectory represents the SetupSystemDirectory task.
+func SetupSystemDirectory(runtime.Sequence, any) (runtime.TaskExecutionFunc, string) {
```

Comment:
move (?) to https://github.com/siderolabs/talos/blob/a5925066c5bf400f554523ffe53b1af6dfb80daa/internal/app/machined/pkg/startup/startup.go#L39-L50